packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -85,14 +85,23 @@ class Amplitude {
 
     // rudderElement.message.context will always be present as part of identify event payload.
     const { traits } = rudderElement.message.context;
-    const { userId } = rudderElement.message;
-
+    const { userId, integrations } = rudderElement.message;
+    const amplitudeIntgConfig = getDestinationOptions(integrations);
```

Comment:
Hi Anant, curious to know about what options we can get from here. Also as I see we are not adding any config here, then how these fieldsToUnset is to be populated.

Reply:
got clarified from @anantjain45823 that it fetched from integrations object